flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorITCase.java

```diff
@@ -531,6 +531,30 @@ public void testLocationWithComma() throws Exception {
         }
     }
 
+    @Test
+    public void testReadHiveDataWithEmptyMapForHiveShim20X() throws Exception {
+        TableEnvironment tableEnv = getTableEnvWithHiveCatalog();
+        tableEnv.getConfig()
+                .getConfiguration()
+                .set(HiveOptions.TABLE_EXEC_HIVE_FALLBACK_MAPRED_READER, true);
+        try {
+            tableEnv.executeSql("create table src(x map<string, string>) stored as parquet");
+            HiveTestUtils.createTextTableInserter(hiveCatalog, "default", "src")
```

Comment:
`HiveTestUtils.createTextTableInserter` is only meant to insert data for text tables. I don't think it works for parquet tables.